.travis.yml

```diff
@@ -0,0 +1,53 @@
+language: android
+dist: trusty
+
+before_cache:
+  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+
+cache:
+  directories:
+    - $HOME/.gradle/caches/
+    - $HOME/.gradle/wrapper/
+
+jdk:
+  - oraclejdk8
```

Comment:
should we use a different jdk vendor?

Reply:
We don't need too. Oracle is still the standard provider of jdk8. And to be honest, it's so tricky to make it work that I would recommend to just keep this setting.